src/test/java/nl/tudelft/ti2206/group9/gui/EndToEndTest.java

```diff
@@ -364,13 +380,6 @@ private void pausePopup(final int buttonNo) {
         sleep(LONG);
     }
 
-    private void playerDies() {
-        Track.getInstance().getPlayer().die();
-        sleep(SLEEP_FACTOR * InternalTicker.NANOS_PER_TICK / InternalTicker.E6);
-        letPlayerSurvive();            // Make sure there are no obstacles
-        sleep(LONG);
-    }
-
     private void clickPopup(final int buttonNo) {
         if (AbstractScene.getPopup() != null) {
```

Comment:
This line isn't modified, but a test generally should not contain an if-statement. This branch is perfect to refactor this: in my opinion you should either remove this or add an else statement with a failure.

< testing, review